src/components/wavyWave/wave.js

```diff
@@ -1,3 +1,28 @@
+import { createElement } from "../../utils/elements";
+// Animation with setInterval() Reference: https://developer.mozilla.org/de/docs/Web/API/WindowOrWorkerGlobalScope/setInterval
+let IntervallID;
+let isPlaying = false;
+function startAnimation() {
+  if (!isPlaying) {
+    changeHeightAndColor();
+    IntervallID = setInterval(changeHeightAndColor, 500);
+    console.log("animation started");
+    isPlaying = true;
+  } else {
+    return;
+  }
+}
+function stopAnimation() {
+  clearInterval(IntervallID);
+  isPlaying = false;
+  console.log("animation stopped");
+}
```

Comment:
you need only one function which did both, try it out
```suggestion

function playStopButton() {
  const status = document.getElementById("loopBtn");

  if (status.innerHTML === "Stop"){
    status.innerHTML = "Start";
    clearInterval(IntervallID);
  } else {
    IntervallID = setInterval(changeHeightAndColor, 500);
    status.innerHTML = "Stop";
  };
  status.classList.toggle("stop-button");
};
```